frontend/simple/components/SupportHistory.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+  <section>
+    <h3 class="title"><i18n>Support History</i18n></h3>
+    <p v-if="history.length === 0">Your group is still in its first month.</p>
+    <div class="history columns" v-else>
+      <div v-for="(percentage, index) in history" class="column is-2">
+        <div :class="['period', getResult(percentage)]">
+          <p class="period-title">{{ months[index] }}</p>
+          <p class="period-txt">{{ Math.floor(percentage * 100) }}%</p>
```

Comment:
we will have a nice filter to format percentage values once the #355 sbp pr is accepted, leaving this here so I don't forget to apply that here once this is merged :)